app/riot/src/client/preview/index.ts

```diff
@@ -1,6 +1,7 @@
 import { start } from '@storybook/core/client';
 
 import './globals';
+// @ts-ignore
```

Comment:
There are some miscellaneous things with `tag2` across many files, it's imported from `riot` (but isn't available is the type), sometimes it's just stored in an unused `const` but if you remove them everything breaks 🧙🏻